tests/unit/test_data_transformer.py

```diff
@@ -52,155 +54,166 @@ def test___fit_continuous_(self, MockBGM):
             output_info=[SpanInfo(2, 'softmax')],
             output_dimensions=2
         )
 
+        data = pd.DataFrame({
+            'x': np.random.random(size=100),
+            'y': np.random.choice(['yes', 'no'], size=100)
+        })
+
+        # Run
         transformer.fit(data, discrete_columns=['y'])
 
+        # Assert
         transformer._fit_discrete.assert_called_once()
         transformer._fit_continuous.assert_called_once()
         assert transformer.output_dimensions == 6
 
-    def test__transform_continuous(self):
-        """Test '_transform_continuous'.
-
-        The `transform_continuous` method first computes the probability that the
-        continuous value came from each component, then samples a component based
-        on that probability and finally returns a (value, onehot) tuple, where the
-        onehot vector indicates the component that was selected and the value
-        is a normalized representation of the continuous value based on the mean
-        and std of the selected component.
-
-        This test mocks the gaussian mixture model used to compute the probabilities
-        as well as the sampling method; it returns deterministic values to avoid
-        randomness in the test. Then, it tests to make sure the probabilities
-        are computed correctly and that the normalized value is computed correctly.
+    @patch('ctgan.data_transformer.BayesGMMTransformer')
+    def test__transform_continuous(self, MockBGM):
+        """Test ``_transform_continuous``.
 
         Setup:
-            - Create column_transform_info with mocked transformer
-            - Mock the BayesianGaussianMixture transformer
-               - specify means, covariances, predict_proba
-               - means = [0, 10]
-               - covariances = [1.0, 11.0]
-            - Mock np.random.choice to choose maximum likelihood
+            - Mock the ``BayesGMMTransformer`` with the transform method returning
+            some dataframe.
+            - Create ``DataTransformer``.
 
         Input:
-            - column_transform_info
-            - raw_column_data = np.array([0.001, 11.9999, 13.001])
+            - ``ColumnTransformInfo`` object.
+            - A dataframe containing a continuous column.
 
         Output:
-            - normalized_value (assert between -1.0, 1.0)
-              - assert approx = [0.0, -1.0, 1.0]
-            - onehot (assert that it's a one-hot encoding)
-              - assert = [[0, 1], [1, 0], [1, 0]]
-
-        Side Effects:
-            - assert predict_proba called
-            - assert np.random.choice with appropriate probabilities
+            - A np.array where the first column contains the normalized part
+            of the mocked transform, and the other columns are a one hot encoding
+            representation of the component part of the mocked transform.
         """
+        # Setup
+        bgm_instance = MockBGM.return_value
+        bgm_instance.transform = Mock()
```

Comment:
Since `bgm_instance` is a mock, I think any methods it has are already mocks, so I don't think this line is necessary.